src/views/Streamer.vue

```diff
@@ -154,11 +154,13 @@
     :roomPassword="room_password"
     @sessionId="onSessionId"
     @viewerCount="onViewerCount"
+    @idTaken="onIdTaken"
   )
   .col-md-1-2
     LogoSvg#logo
     #live-indicator(:class="{ live: isSharingOn && sessionId }") LIVE
   .col-md-1-2
+    div#id-taken(v-if="useridAlreadyTaken") {{useridAlreadyTaken}} already taken! 
```

Comment:
Can we make this text red so it's more noticeable? 
Also, let's use the text from above `Whoops, useridAlreadyTaken + ' already taken. Please choose another room name.'`

Reply:
I've gone and added some styling to the alert.